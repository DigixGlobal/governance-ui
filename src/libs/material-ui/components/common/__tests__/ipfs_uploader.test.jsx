import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import IpfsUploader from '../ipfs_uploader';

describe('<IpfsUploader />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <IpfsUploader
        name="Ipfs test"
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
