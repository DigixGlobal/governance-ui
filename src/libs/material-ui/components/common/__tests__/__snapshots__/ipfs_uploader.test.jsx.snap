// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IpfsUploader /> renders correctly 1`] = `
<FormField>
  <label
    htmlFor="upload"
  >
    Upload to IPFS
  </label>
  <FormGroup
    widths="equal"
  >
    <FormField>
      <Button
        as="button"
        color="green"
        content="Browse File"
        disabled={false}
        fluid={true}
        icon="upload"
        onClick={[Function]}
      />
      <input
        onChange={[Function]}
        style={
          Object {
            "position": "absolute",
            "visibility": "hidden",
          }
        }
        type="file"
      />
    </FormField>
  </FormGroup>
</FormField>
`;
